sasview/setup_exe.py

```diff
@@ -292,9 +296,29 @@ def dll_check(dll_path, dlls):
         data_files.append(("test\\save_states", [f]))
 
 # Copying the sample data user data
-for f in findall(test_upcoming_dir):
+for f in findall(test_sesans_dir):
     if not ".svn" in f:
-        data_files.append(("test\\upcoming_formats", [f]))
+        data_files.append(("test\\sesans_data", [f]))
+
+# Copying the sample data user data
+for f in findall(test_convertible_dir):
+    if not ".svn" in f:
```

Comment:
I copied and pasted the blocks without looking at that line. That's a remnant from our svn days that can (with almost 100% surety) be removed. I'll do that tomorrow.